test/index.js

```diff
@@ -6,22 +6,27 @@ const fs = require('fs')
 const c = require('../constants')
 const events = require('../events')
 
-const env = process.env
-
-if (!env.DC_ADDR || !env.DC_MAIL_PW) {
-  throw new Error('No credentials. Please specify $DC_ADDR and $DC_MAIL_PW !')
-}
-
 let dc = null
 
 test('setUp dc context', t => {
   t.plan(3)
   dc = new DeltaChat({
-    addr: env.DC_ADDR,
-    mail_pw: env.DC_MAIL_PW,
+    addr: 'delta1@delta.localhost',
```

Comment:
Since we're not using encrypted environment variables on Travis anymore it means external pull requests should also work since we're no longer requiring keeping them secret since we're running locally.